src/pages/team/languages.json

```diff
@@ -64,53 +64,79 @@
     ],
     "ppmc": [
       {
-        "apacheId": "",
+        "apacheId": "kirs",
         "avatarUrl": "https://avatars.githubusercontent.com/u/16631152?v=4",
         "email": "kirs@apache.org",
         "gitUrl": "https://github.com/CalvinKirs",
         "githubId": "CalvinKirs",
         "name": "Calvin Kirs"
       },
       {
-        "apacheId": "",
+        "apacheId": "rickyhuo",
         "avatarUrl": "https://avatars.githubusercontent.com/u/18550072?v=4",
-        "email": "",
+        "email": "rickyhuo@apache.org",
         "gitUrl": "https://github.com/RickyHuo",
         "githubId": "RickyHuo",
         "name": "Ricky Huo"
       },
       {
-        "apacheId": "",
+        "apacheId": "gaoyingju",
         "avatarUrl": "https://avatars.githubusercontent.com/u/2291859?v=4",
-        "email": "",
+        "email": "gaoyingju@apache.org",
         "gitUrl": "https://github.com/garyelephant",
         "githubId": "garyelephant",
         "name": "Gary Gao"
       },
       {
-        "apacheId": "",
+        "apacheId": "ridxiong",
         "avatarUrl": "https://avatars.githubusercontent.com/u/41584238?v=4",
-        "email": "lidongdai@apache.org",
+        "email": "ridxiong@apache.org",
         "gitUrl": "https://github.com/kid-xiong",
         "githubId": "kid-xiong",
         "name": "Kid Xiong"
       },
       {
-        "apacheId": "",
+        "apacheId": "lidongdai",
         "avatarUrl": "https://avatars.githubusercontent.com/u/53861852?v=4",
         "email": "lidongdai@apache.org",
         "gitUrl": "https://github.com/nielifeng",
         "githubId": "nielifeng",
         "name": "LiFeng Nie"
       },
       {
-        "apacheId": "",
+        "apacheId": "zhongjiajie",
         "avatarUrl": "https://avatars.githubusercontent.com/u/15820530?v=4",
         "email": "zhongjiajie@apache.org",
         "gitUrl": "https://github.com/zhongjiajie",
         "githubId": "zhongjiajie",
         "name": "Jiajie Zhong"
       }
+    ],
+    "committer": [
+      {
+        "apacheId": "leo65535",
+        "avatarUrl": "https://avatars.githubusercontent.com/u/95013770?v=4",
+        "email": "leo65535@apache.org",
+        "gitUrl": "https://github.com/leo65535",
+        "githubId": "leo65535",
+        "name": "Lei Xie"
+      },
+      {
+        "apacheId": "benjobs",
+        "avatarUrl": "https://avatars.githubusercontent.com/u/13284744?v=4",
+        "email": "benjobs@apache.org",
+        "gitUrl": "https://github.com/wolfboys",
+        "githubId": "wolfboys",
+        "name": "Huajie Wang"
+      },
+      {
+        "apacheId": "wuchunfu",
+        "avatarUrl": "https://avatars.githubusercontent.com/u/29206593?v=4",
+        "email": "wuchunfu@apache.org",
+        "gitUrl": "https://github.com/wuchunfu",
+        "githubId": "wuchunfu",
+        "name": "ChunFu Wu"
+      }
```

Comment:
```suggestion
      },
      {
        "apacheId": "yangxiang",
        "avatarUrl": "https://avatars.githubusercontent.com/u/8814185?v=4",
        "email": "yangxiang@apache.org",
        "gitUrl": "https://github.com/yx91490",
        "githubId": "yx91490",
        "name": "Yang Xiang"
      }
```

Reply:
We have a new committer @yx91490 . Please make sure your information is correct.

Reply:
BTW, we can see all our committer and PPMC mentor in https://whimsy.apache.org/roster/

Reply:
@zhongjiajie Thanks for the reminder.